student/pom.xml

```diff
@@ -24,7 +24,7 @@
 	<properties>
 		<progman-client.version>3.0.1</progman-client.version>
 		<sb11-mna-client.version>3.0.0</sb11-mna-client.version>
-		<tds-exam-client.version>0.0.14</tds-exam-client.version>
+		<tds-exam-client.version>0.0.15-SNAPSHOT</tds-exam-client.version>
```

Comment:
Will update this to use the release version of `tds-exam-client` when [this PR against TDS_ExamService](https://github.com/SmarterApp/TDS_ExamService/pull/174) is approved.